runtime/queries/metricsview.go

```diff
@@ -696,6 +712,14 @@ func metricsViewDimensionColumn(dimension *runtimev1.MetricsViewSpec_DimensionV2
 	return dimension.Name
 }
 
+func metricsViewDimensionExpression(dimension *runtimev1.MetricsViewSpec_DimensionV2) string {
+	if dimension.Expression != "" {
+		return dimension.Expression
+	}
+	// if there is expression treat as a column reference
+	return safeName(metricsViewDimensionColumn(dimension))
```

Comment:
If possible, let's inline `metricsViewDimensionColumn` into `metricsViewDimensionExpression` to not have too much ambiguity

Reply:
Makes sense. There is one usage in older filters code which i will remove and merge these.